frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.styled.tsx

```diff
@@ -103,9 +101,30 @@ const footerVariantStyles = {
   `,
 };
 
-export const ParametersWidgetContainer = styled(FullWidthContainer)`
+export const ParametersWidgetContainer = styled(FullWidthContainer)<{
+  hasScroll: boolean;
+  isSticky: boolean;
+}>`
   padding-top: ${space(1)};
   padding-bottom: ${space(1)};
+
+  ${props =>
+    props.hasScroll &&
+    css`
+      border-bottom: 1px solid ${color("border")};
+    `}
+
+  ${props =>
+    props.isSticky &&
+    css`
+      position: sticky;
+      top: 0;
+      left: 0;
+      width: 100%;
+      z-index: 3;
+
+      background-color: ${color("white")};
```

Comment:
Apologies for not catching this until now, but I just realized the embedded version has configurable color themes. We need to account for this in the parameters container, so that we change the color based on the theme

**Light** :heavy_check_mark: 
![image](https://github.com/metabase/metabase/assets/22608765/43ec23ae-f9bb-4b3d-b97d-55833b94adc2)

**Dark** :x: 
![image](https://github.com/metabase/metabase/assets/22608765/24130a90-26ad-4fde-bb86-92e3094caaae)

**Transparent** :x: 
![image](https://github.com/metabase/metabase/assets/22608765/4bdfd58f-0772-4af1-bcb8-016b4e474205)

I think we need to do something special for transparent though, since the sticky filters container will look bad if it is transparent. 

**Transparent Filters** :x:
![image](https://github.com/metabase/metabase/assets/22608765/039c479d-70dc-40d8-8a3b-4d7ad60e2be8)

Reply:
Good catch!

Reply:
Good catch indeed! 🙏

Reply:
@JesseSDevaney please take another look. [`25fcde6946`](https://github.com/metabase/metabase/pull/38766/commits/25fcde6946bdf4898ab9e8f9282aab634bd4bbbd) should fix filter panel colors and [`46fcd66f0f`](https://github.com/metabase/metabase/pull/38766/commits/46fcd66f0f697cfb3d43cbcdccc4e8a00caee9d0) will disable the sticky behavior if a dashboard is embedded with a "transparent" them